dist/humans.txt

```diff
@@ -0,0 +1,15 @@
+# humanstxt.org/
+# The humans responsible & technology colophon
+
+# TEAM
+
+    <name> -- <role> -- <twitter>
```

Comment:
This file doesn't seem finished, yet.

Reply:
I think we can merge it for now and add the members later once we figure out positions and stuff (just leave it as a placeholder for now).

Reply:
I am just not convinced, how this file would be useful for now. I propose that we either fill it with contents remove it for now, until we have contents. 

It may not be problematic in terms of space, however I think it is confusing. If I see such a file, I would expect it to have useful contents.

Reply:
Ok lets remove this - it's not a big deal.